spec/models/invenio_rdm_record_converter_spec.rb

```diff
@@ -310,6 +311,40 @@
     end
   end
 
+  describe "#file_permissions" do
+    let(:role) { Role.create(name: 'export_editor') }
+    let(:exporter) { FactoryGirl.create(:user, username: "exp987") }
+    before do
+      exporter.add_role(role.name)
+      generic_file.permissions.create!(
+        name: role.name, type: 'group', access: 'edit',
+        access_to: generic_file.id)
+    end
+
+    context "with role" do
+      before do
+        role_store.build_role_store_data
+      end
+
+      let(:expected_permissions) {
+        {
+          "read": ["public"],
+          "edit": ["usr1234", "exp987"]
```

Comment:
Shouldn't this be failing unless we change to

```
"edit": [
  {"netid": "usr1234", "email": <email>},
  {...}
]
```
?

Reply:
No, this is right. The netid is the key, and that's all we are getting here. Did we want to include email in the individual record json? Email is being included in the created `tmp/export/roles_data.json`.

Reply:
After some thinking, yes I think having the email in the exported json as well would be best.

1- we already do so for the owner, so it's nice to have the same pattern.
2- would need to parse and remap the role_data.json to get a general netid to email mapping in the import

So having something like:

```
"edit": [ {"usr1234": <email>},  {...} ]
```
(same for read)

in the record would be good.

Reply:
Okay, thank you for clarifying. The output for permissions here is now: 
```
"read": {
  "public": ""
},
"edit": {
  user.username => user.email
}
```

Reply:
I am sorry about this., I don't think I was explicit enough about the email+netid :cry:  
I meant that the generated users (from group transformation or not) should be exported with their email and netid _in all cases_. The transformation of groups into netids has highlighted how the import may receive just
some netids for users (in general), but creating a user on the import side requires an email.

We could put this in a separate task: it might be faster to do 1 query to get all netid+emails (if there is a User model) and then do the mapping everywhere. In other words, if a user was given direct access in the edit, also providing its email would be necessary. 

Second, will the difference between the colon notation and the rocket notation affect the
output? 

this:

```ruby
"owner": {
  "netid": user.username,
  "email": user.email
},
```

that:

```ruby
"edit": {
  "<netid>" => <email>
}
```